render/wgpu/shaders/blend/alpha.wgsl

```diff
@@ -12,8 +12,8 @@ struct VertexOutput {
 
 @vertex
 fn main_vertex(in: common__VertexInput) -> VertexOutput {
-    let pos = common__globals.view_matrix * transforms.world_matrix * vec4<f32>(in.position.x, in.position.y, 1.0, 1.0);
-    let uv = vec2<f32>((pos.x + 1.0) / 2.0, -((pos.y - 1.0) / 2.0));
+    let pos = common__globals.view_matrix * transforms.world_matrix * vec4<f32>(in.position.x, in.position.y, 0.0, 1.0);
+    let uv = vec2<f32>(in.position.xy);
```

Comment:
Unsure on this change to be honest. Because in other WGSL files, the uv is independent from world_matrix (and sometimes uses another matrix `texture_matrix`), I thought this is correct, but not sure.

Reply:
I think this is correct. I believe Ruffle uses those coordinate systems?:
* `in.position`: `[0, 1] * [0, 1]` (e.g. `Quad`) (wgpu texture coordinate system)
* `world_matrix * in.position`: `[0, width] * [0, height]` (FP coordinate system)
* `view_matrix * world_matrix * in.position`: `[-1, 1] * [-1, 1]` (wgpu render coordinate system)

wgpu coordinate system: https://github.com/gfx-rs/wgpu?tab=readme-ov-file#coordinate-systems